list.go

```diff
@@ -114,19 +115,19 @@ func (s *ItemList) Keys() []string {
 func (s *ItemList) GetItem(key string) (Item, error) {
 
 	if s == nil {
-		return Item{}, fmt.Errorf("configstore: get '%s': non-initialized item list", key)
+		return Item{}, errors.NotAssignedf("configstore: get '%s': non-initialized item list", key)
 	}
 
 	l := (&ItemFilter{}).Slice(key).Apply(s)
 
 	switch len(l.Items) {
 	case 0:
-		return Item{}, fmt.Errorf("configstore: get '%s': no item found", key)
+		return Item{}, errors.NotFoundf("configstore: get '%s': no item found", key)
 	case 1:
 		return l.Items[0], nil
 
 	}
-	return Item{}, fmt.Errorf("configstore: get '%s': ambiguous, %d items share that key", key, len(l.Items))
+	return Item{}, errors.AlreadyExistsf("configstore: get '%s': ambiguous, %d items share that key", key, len(l.Items))
```

Comment:
AlreadyExists on a getter ? It's more some kind of NonUnique but doesn't exists in juju :/